packs/actions/fight-with-fear.json

```
@@ -11,7 +11,7 @@
         },
         "category": "interaction",
         "description": {
-            "value": "<p><strong>Trigger</strong> A creature that you can see uses a mental effect against you</p>\n<hr />\n<p><strong>Effect</strong> The triggering creature must attempt a Will save, which has the following effects.</p>\n<hr />\n<p><strong>Success</strong> The creature is unaffected.</p>\n<p><strong>Failure</strong> The creature is @UUID[Compendium.pf2e.conditionitems.Item.Frightened]{Frightened 2}, and you gain a +2 status bonus to your saving throw or other defense against the triggering mental effect.</p>\n<p><strong>Critical Failure</strong> The creature is frightened 2, and you're unaffected by the triggering mental effect.</p>"
+            "value": "<p><strong>Trigger</strong> A creature that you can see uses a mental effect against you</p><hr /><p><strong>Effect</strong> The triggering creature must attempt a @Check[type:will] save, which has the following effects.</p><hr /><p><strong>Success</strong> The creature is unaffected.</p>\n<p><strong>Failure</strong> The creature is @UUID[Compendium.pf2e.conditionitems.Item.Frightened]{Frightened 2}, and you gain a +2 status bonus to your saving throw or other defense against the triggering mental effect.</p>\n<p><strong>Critical Failure</strong> The creature is frightened 2, and you're unaffected by the triggering mental effect.</p>"
```

```suggestion
            "value": "<p><strong>Trigger</strong> A creature that you can see uses a mental effect against you</p>\n<hr />\n<p><strong>Effect</strong> The triggering creature must attempt a will save, which has the following effects.</p>\n<hr />\n<p><strong>Success</strong> The creature is unaffected.</p>\n<p><strong>Failure</strong> The creature is @UUID[Compendium.pf2e.conditionitems.Item.Frightened]{Frightened 2}, and you gain a +2 status bonus to your saving throw or other defense against the triggering mental effect.</p>\n<p><strong>Critical Failure</strong> The creature is frightened 2, and you're unaffected by the triggering mental effect.</p>"
```
This is against your spell DC and the spell itself would be the better way to do the save since it carries the traits and DC properly.